### DISCUSSION TITLE
反馈一个Bug

### DISCUSSION THREAD

**Discussion by luoliguang** [Quorafind/Obsidian-Thino#634] [Q&A 问答]
![image](https://github.com/Quorafind/Obsidian-Thino/assets/74090098/bc9c8965-574a-4f36-b497-a3447e796f2c)
Thion手机微信上同步到Obsidian 作为单文件同步到日记中的时候日期的排序不准确。比如今天27号 前面有26、25、24但是它会自动插入到21号后面，不会在26号后面。

我不知道是我的设置问题还是一个Bug呢

**Comment by juestchaos** [comment]
问题放到 issue 中提交
